packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -98,9 +73,17 @@ export async function processOutboundDocumentQueryResps({
       },
     });
 
-    const docRefsWithMetriportId = filterDocRefsWithMetriportId(docRefs);
+    const responseWithDocsToDownload = await getRespWithDocsToDownload({
+      cxId,
+      patientId,
+      requestId,
+      response,
+    });
+
+    const docsToDownload = responseWithDocsToDownload.flatMap(
+      result => result.documentReference ?? []
```

Comment:
nit: `documentReference` can't be undefined, so we don't need the fallback, right?

Reply:
Yea if the response is a failure then documentReference wont be present

Reply:
That's odd, I had this branch checked out and VSCode was telling me it was always present, but maybe the TSServer didn't update properly after the checkout and I see your point. NABD.